### ISSUE TITLE
Add alarms to Timeline

### ISSUE THREAD

**Issue by martinpaljak** [SeaPea/GentleWake#8]

**Comment by SeaPea** [comment]
This has been requested a few times, but currently it requires communication with phone-side code, which then calls a web service to add a timeline pin. I am holding out, hoping Pebble adds a local API call on the watch to add timeline pins as otherwise that is a lot of code for just a nice to have feature.

**Comment by martinpaljak** [comment]
Indeed. Has there been some "official feature request" for the local API? Where does one post such requests?

**Comment by martinpaljak** [comment]
And I love that there is no need for a phone configuration! While the "cloud watch" concept has its merits, I'd expect basic features (that have to do with time and alarms etc) be functional _without_ a connection.

**Comment by clach04** [comment]
https://rebble.io/2019/07/24/its-timeline-time.html
